From 2252bec27ecd3d0353d1868af5a5f325450cc1d2 Mon Sep 17 00:00:00 2001
From: dann frazier <dann.frazier@canonical.com>
Date: Thu, 21 May 2015 10:28:48 -0600
Subject: arm64/setjmp: Add missing license macro

Including the setjmp module in an arm64-efi image will cause it to
immediately exit with an "incompatible license" error.

The source file includes a GPLv3+ boilerplate, so fix this by declaring a
GPLv3+ license using the GRUB_MOD_LICENSE macro.

Signed-off-by: dann frazier <dann.frazier@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1459871
Origin: upstream, http://git.savannah.gnu.org/cgit/grub.git/commit/?id=3ac342205dc81293bb8e2d91b8c5ebe124b4ad35

Patch-Name: arm64-setjmp-Add-missing-license-macro.patch
---
 grub-core/lib/arm64/setjmp.S | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/grub-core/lib/arm64/setjmp.S b/grub-core/lib/arm64/setjmp.S
index adaafe4..eabfd99 100644
--- a/grub-core/lib/arm64/setjmp.S
+++ b/grub-core/lib/arm64/setjmp.S
@@ -17,8 +17,10 @@
  */
 
 #include <grub/symbol.h>
+#include <grub/dl.h>
 
 	.file	"setjmp.S"
+GRUB_MOD_LICENSE "GPLv3+"
 	.text
 
 /*
